client/src/pages/Medical.js

```diff
@@ -64,30 +181,48 @@ export const Medical = () => {
   useEffect(initializeMedicalState, []);
 
   return (
-    <div className="text-center mx-auto" style={CONTAINER_STYLES}>
-      <h5>Local Emergency Number</h5>
+    <div className="text-center">
+      <h3>Your Local Emergency Number</h3>
       <h1 className="text-primary display-4 font-weight-bolder">
         {state.loading
           ? "Loading..."
           : state.emergencyNumber || state.errorMessage}
       </h1>
-      <div style={{ display: "flex", margin: "30px 10px 10px 50px" }}>
-        <Link
-          to="/nearest-hospital"
-          className="btn btn-outline-info mb-3 mr-4 float-left"
-        >
-          Nearest Hospitals
-        </Link>
-        <Link
-          to="/symptoms-check"
-          className="btn btn-outline-info mb-3 mr-4 float-left"
-        >
-          Symptoms check
-        </Link>
-        <Link to="/find-help" className="btn btn-outline-info mb-3 float-left">
-          Find Help
-        </Link>
-      </div>
+      <StyledWelcome level={3} size="xlarge">
+        Please Call Your Local Emergency Number
+      </StyledWelcome>
+      <OnboardingContainer style={{ display: "flex", flexWrap: "wrap" }}>
+        <FlexChild>
+          <ImageButton
+            type="ghost"
+            inactiveImg={nearestHospitalUnselected}
+            activeImg={nearestHospitalSelected}
+            onClick={() => props.history.push("/nearest-hospital")}
```

Comment:
Maybe this is out of scope of this pull request
but should these be like "tabs" inside the "medical" page or change the full "page" ?